modules/msk/netmhcstabpan/main.nf

```diff
@@ -0,0 +1,56 @@
+process NETMHCSTABPAN {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'docker://mskcc/netmhctools:1.0.0':
+        'docker.io/mskcc/netmhctools:1.0.0' }"
+
+    input:
+    tuple val(meta), path(inputFasta), val(hlaString), val(inputType)
+
+
+    output:
+    tuple val(output_meta), path("*.netmhcstabpan.output"),   emit: netmhcstabpanoutput
+    path "versions.yml",                                      emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def hla = hlaString.trim()
+    output_meta = meta.clone()
+    output_meta.typeMut = inputType == "MUT" ? true : false
```

Comment:
it isnt exactly sample specific, each sample is run through a module twice.  Once with MUT and once with WT.   These values differentiate the two items in the channel.  This could also happen with a meta data map perhaps.  We can explore it but I think its okay for now